core/kazoo_config/src/kapps_config_util.erl

```diff
@@ -49,9 +49,9 @@ load_config_from_reseller(Account, Config) ->
     ResellerId = kz_services:find_reseller_id(Account),
     load_config_from_account(ResellerId, Config).
 
--spec load_config_from_system(api_binary(), ne_binary()) -> {ok, kz_json:object()} | {error, any()}.
+-spec load_config_from_system(api_binary(), ne_binary()) -> {ok, kz_json:object()}.
 load_config_from_system(_Account, Config) ->
-    {'ok', kz_json:get_value(<<"default">>, maybe_new(kapps_config:get_category(Config)))}.
+    {'ok', kz_json:get_value(<<"default">>, kapps_config:get_category(Config), kz_json:new())}.
```

Comment:
@lazedo Hi!

Erlang/OTP 18 [erts-7.2] [source] [64-bit] [smp:4:4] [async-threads:10] [kernel-poll:false]

Eshell V7.2  (abort with ^G)
(kazoo_apps@sip.okui.info)1> kapps_config_util:get_config(<<"681d0ba47ea33e6a48233e1403679a2c">>, <<"paypal">>).
** exception error: no function clause matching kz_json:merge(#Fun<kz_json.17.107100778>,
                                                              {[{<<"enabled">>,true},{<<"_id">>,<<"configs_paypal">>}]},
                                                              undefined) (src/kz_json.erl, line 289)
     in function  lists:foldl/3 (lists.erl, line 1262)
     in call from kapps_config_util:get_config/3 (src/kapps_config_util.erl, line 33)
     in call from kapps_config_util:get_config/2 (src/kapps_config_util.erl, line 26)


It's easy:
- load_config_from_system(_Account, Config) gives {ok, 'undefined'};
- maybe_new/2 passes 'undefined' as is, without converting it to JObj;
- kz_json:merge/1 fails because it can't handle 'undefined'

(kazoo_apps@sip.okui.info)1> Confs = [{[{<<"_id">>,<<"configs_paypal">>},{<<"_rev">>,<<"5-62ee1919d466c42d02824a8529dd8235">>},{<<"environment">>,<<"sandbox">>},{<<"hosted_button_id">>,<<"9YN34FUZYDLYQ">>},{<<"currency">>,<<"RUB">>},{<<"reciever_id">>,<<"JBVTF9RFKREWL">>}]},{[{<<"_id">>,<<"configs_paypal">>},{<<"_rev">>,<<"5-62ee1919d466c42d02824a8529dd8235">>},{<<"environment">>,<<"sandbox">>},{<<"hosted_button_id">>,<<"9YN34FUZYDLYQ">>},{<<"currency">>,<<"RUB">>},{<<"reciever_id">>,<<"JBVTF9RFKREWL">>}]},undefined,{[{<<"enabled">>,true},{<<"_id">>,<<"configs_paypal">>}]}].
[{[{<<"_id">>,<<"configs_paypal">>},
   {<<"_rev">>,<<"5-62ee1919d466c42d02824a8529dd8235">>},
   {<<"environment">>,<<"sandbox">>},
   {<<"hosted_button_id">>,<<"9YN34FUZYDLYQ">>},
   {<<"currency">>,<<"RUB">>},
   {<<"reciever_id">>,<<"JBVTF9RFKREWL">>}]},
 {[{<<"_id">>,<<"configs_paypal">>},
   {<<"_rev">>,<<"5-62ee1919d466c42d02824a8529dd8235">>},
   {<<"environment">>,<<"sandbox">>},
   {<<"hosted_button_id">>,<<"9YN34FUZYDLYQ">>},
   {<<"currency">>,<<"RUB">>},
   {<<"reciever_id">>,<<"JBVTF9RFKREWL">>}]},
 undefined,
 {[{<<"enabled">>,true},{<<"_id">>,<<"configs_paypal">>}]}]
(kazoo_apps@sip.okui.info)2> kz_json:merge(lists:reverse(Confs)).
** exception error: no function clause matching kz_json:merge(#Fun<kz_json.17.107100778>,
                                                              {[{<<"enabled">>,true},{<<"_id">>,<<"configs_paypal">>}]},
                                                              undefined) (src/kz_json.erl, line 289)
     in function  lists:foldl/3 (lists.erl, line 1262)
(kazoo_apps@sip.okui.info)3>